Bottom: 3c14299067df708af8917cb56dd67e309d06b679
Top:    ca7e9cfe2c5c9c76c2477569047c8430973713a9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 00:45:13 +0000

Refresh of hid-api-make-hid-prompt_for-re

---

diff --git a/src/action.c b/src/action.c
index 75d56ad..9e1370b 100644
--- a/src/action.c
+++ b/src/action.c
@@ -7606,7 +7606,7 @@ ActionImport (int argc, char **argv, int x, int y)
 
   if (mode && strcasecmp (mode, "setdisperse") == 0)
     {
-      const char *ds, *units;
+      char *ds, *units;
       char buf[50];
 
       ds = ARG (1);
